driver/desktop/key.go

```diff
@@ -48,6 +48,18 @@ const (
 	SuperModifier = fyne.KeyModifierSuper
 )
 
+// Action represents the change of state of a key or mouse button event
+type Action int
+
+const (
+	// Release Keyboard button was released
+	Release Action = 0
```

Comment:
Why was it necessary to add new public APIs? (and why desktop specific?)

Reply:
It looks like these are only used in the internal desktop driver code. If required, that is where they should be...